### DISCUSSION TITLE
Image analysis timing out with "failed to open: failed to copy: read tcp / read: operation timed out"

### DISCUSSION THREAD

**Discussion by varanasikalyan** [aquasecurity/trivy#5088] [Bugs]
### Description

Trivy image scan notably on very big images with one or more layers having big size are failing at analysis step with "failed to open: failed to copy: read tcp / read: operation timed out"

### Desired Behavior

I expected the scan should continue even when there is an err while opening the file.

![image](https://github.com/aquasecurity/trivy/assets/45425247/89d04b01-100d-4ac9-a2bf-eecce35124e9)

### Actual Behavior

scan failed with an err stating couldn't open file
![image](https://github.com/aquasecurity/trivy/assets/45425247/8b97b6c3-7c3a-4845-9513-593b1c812289)

### Reproduction Steps

```bash
This issue happens when we perform image scan for a very large image with large layers.
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
"failed to open: failed to copy: read tcp / read: operation timed out"
```

### Operating System

linux

### Version

```bash
0.43.1
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by AnaisUrlichs** [comment]
Where is that screenshot taken from?

**Reply by brendakleiman** [reply]
This is from the UI when manually pulling the image.
